src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,168 @@
+package duke;
+
+import duke.commands.BaseCommand;
+import duke.commands.CommandResult;
+import duke.commands.ErrorCommand;
+import duke.commands.ExitCommand;
+import duke.commands.tasks.BaseTaskCommand;
+import duke.exceptions.IncorrectArgumentException;
+import duke.exceptions.InvalidDateTimeException;
+import duke.exceptions.InvalidTaskSpecificationException;
+import duke.exceptions.MissingArgumentException;
+import duke.exceptions.NoCommandException;
+import duke.exceptions.UnknownCommandException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * The type duke.Duke.
+ */
+public class Duke {
+
+    private static final String defaultFilePath = "src/main/java/duke/data/tasks.txt";
```

Comment:
Perhaps the naming of this constants should be in all uppercase?

Reply:
Hmm I see, thank you for pointing this out